src/Microsoft.CSharp/src/MultilingualResources/Microsoft.CSharp.de.xlf

```diff
@@ -0,0 +1,520 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I suspect that many of the translation files will need manual editing, because the machine translations typically aren't that great, as Alexander noted is the case with these translations.  So to get much value out of this approach, I think we'd have to support the ability to provide override content and otherwise dynamically use machine translations.

I see a few drawbacks to that approach:
- The machine translations could change over time if they're being generated dynamically.  I think MAT uses an online service like Bing Translate or Google Translator, and if the translations change, then something that was reviewed and considered correct initially may become incorrect later without us knowing.
- It would make it more difficult for people to review the translations if they're not checked in.  Rather than just reviewing the translations in source control, they would have to locally generate the machine translations themselves.
- Setting up the infrastructure to generate the .xlf files during the build using dynamically-generated machine translations combined with the override content would require substantial investment.  As a rough initial estimate I would put it at 2 weeks, but it could even be more than that.

I agree that the translations are a lot of code, but checking in translations is standard practice (as far as I know), and is what we do in the main .NET Framework code in TFS.